packages/cli/src/commands/pg/promote.ts

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable complexity */
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import heredoc from 'tsheredoc'
+import {getAttachment, getRelease} from '../../lib/pg/fetcher'
+import pgHost from '../../lib/pg/host'
+import {PgStatus, PgDatabase} from '../../lib/pg/types'
+
+export default class Promote extends Command {
+  static topic = 'pg';
+  static description = 'sets DATABASE as your DATABASE_URL';
+  static flags = {
+    force: flags.boolean({char: 'f'}),
+    app: flags.app({required: true}),
+    remote: flags.remote(),
+  };
+
+  static args = {
+    database: Args.string({required: true}),
+  };
```

Comment:
Reminder to remove semi-colons. It seems these are introduced by our migration tool. I don't know if it would be useful to change the migration tool at this point to avoid this.